cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -78,16 +78,27 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
       cy.request(`${Cypress.env('currentPath')}.json`).then(({ body }) => {
         const numRelatedContentGroups = body.relatedContent.groups.length;
 
-        if (numRelatedContentGroups > 0) {
-          const assetURI =
-            body.relatedContent.groups[0].promos[0].locators.assetUri;
-          cy.get('li[class^="StoryPromoLi"] > div[class^="StoryPromoWrapper"]')
-            .find('h3')
-            .within(() => {
-              cy.get('a')
-                .should('have.attr', 'href')
-                .and('include', assetURI);
-            });
+        const requestPath = Cypress.env('currentPath');
```

Comment:
This looks good :)

as a side note, you can `return` early from functions - this can avoid the need to have deeply nested conditionals-within-conditionals

eg:

```
if (isLegacyAsset) {
  return cy.log('Test skipped because legacy MAP')
}

if (numRelatedContentGroups === 0) {
  return cy.log('Test skipped because no related content')
}

// We will only get this far if none of the above returns were hit
const assetURI = body.relatedContent.groups[0].promos[0].locators.assetUri;

// ...
```

Reply:
Ah nice. Yea this would be a good solution to eliminate those else(s) {}